vars/odsComponentStageScanWithSonar.groovy

```diff
@@ -1,15 +1,33 @@
 import org.ods.component.ScanWithSonarStage
 import org.ods.component.IContext
 
+import org.ods.services.BitbucketService
 import org.ods.services.SonarQubeService
 import org.ods.services.ServiceRegistry
 
 def call(IContext context, Map config = [:]) {
+    def bitbucketService = ServiceRegistry.instance.get(BitbucketService)
+    if (!bitbucketService) {
+        bitbucketService = new BitbucketService(
```

Comment:
Can we assume it is created by the pipeline already (Pipeline.groovy)?

Reply:
AFAIK this is done this way so it can be tested autonomously